scripts/actions/mobskills/astral_flow_pet.lua

```diff
@@ -42,18 +55,12 @@ mobskillObject.onMobWeaponSkill = function(target, mob, skill)
 
     -- Find proper pet skill
     local petFamily = pet:getFamily()
-    local skillId = 919
+    local skillId   = 919
 
-    if     petFamily == 34 or petFamily == 379 then skillId = 919 -- carbuncle searing light
-    elseif petFamily == 36 or petFamily == 381 then skillId = 839 -- fenrir    howling moon
-    elseif petFamily == 37 or petFamily == 382 then skillId = 916 -- garuda    aerial blast
-    elseif petFamily == 38 or petFamily == 383 then skillId = 913 -- ifrit     inferno
-    elseif petFamily == 40 or petFamily == 384 then skillId = 915 -- leviathan tidal wave
-    elseif petFamily == 43 or petFamily == 386 then skillId = 918 -- ramuh     judgment bolt
-    elseif petFamily == 44 or petFamily == 387 then skillId = 917 -- shiva     diamond dust
-    elseif petFamily == 45 or petFamily == 388 then skillId = 914 -- titan     earthen fury
-    else
-        printf("[astral_flow_pet] received unexpected pet family %i. Defaulted skill to Searing Light.", petFamily)
+    for mobSkillId, petFamilyList in ipairs(petAstralFlowAbility) do
+        if utils.contains(petFamily, petFamilyList) then
+            skillId = mobSkillId
```

Comment:
break?

Reply:
Also, default `skillId` default value is already 919. Seems reduntant to set it to 919 again if a match happens.

Reply:
You're right, this is a carryover from a way it was previously handled